### DISCUSSION TITLE
Missing Shield Icon

### DISCUSSION THREAD

**Discussion by quick007** [tailwindlabs/heroicons#1183] [Ideas]
I'm aware that you don't take icon requests here, but this isn't really a new icon per se. The heroicons logo is a shield, and there are two different shield icons that are available for use:
![image](https://github.com/tailwindlabs/heroicons/assets/69743171/023da481-cae7-4071-80bc-459431552965)

Notably, the actual shield itself (with no icon) seems to be missing. I'm guessing this was an oversight of some sort, hence why it's a bug and not a new icon suggestion.

**Comment by RobinMalfait** [comment]
Hey!

Marked this as an idea instead of a bug because there are other icons like this with the same idea. For example if you look at the circle, triangle or square icons then you will notice that they all have a little additional icon inside. But "square", "triangle" and "circle" do not exist as standalone icons.

**Comment by quick007** [comment]
@RobinMalfait Makes sense! I know the heroicons team normally doesn't take new icon pull requests, but would you consider taking a PR for icons that (more or less) already exist, like the shield and other icons? I'd be happy to make them in all the sizes and use the current icons as a basis, so they wouldn't look out of place or missing.

**Comment by ndri** [comment]
I added `shield` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=shield
